database/migrations/2016_06_15_000000_modify_bills_use_integer.php

```diff
@@ -3,14 +3,16 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
-class ModifyBillsAddInvoice extends Migration {
+class ModifyBillsUseInteger extends Migration {
     public function up() {
         Schema::table('bills', function($table) {
             //Use integer in cents, as it's more precise.
-            $table->dropColumn([
+            $table->dropColumn(array(
                 'amount', 'int_amount',
                 'driver_amount', 'taxes'
-            ]);
+            ));
+        });
+        Schema::table('bills', function($table) {
```

Comment:
It would not migrate the way it was on my system. Many of the style guides I found said that the removal and re-addition of a field should be two separate migrations, but this solves the problem so I don't mind